village/src/components/SmurfForm.js

```diff
@@ -10,21 +11,34 @@ class SmurfForm extends Component {
     };
   }
 
-  addSmurf = event => {
+  addSmurf = (event) => {
     event.preventDefault();
-    // add code to create the smurf using the api
-
-    this.setState({
-      name: '',
-      age: '',
-      height: ''
-    });
+    const smurf = {
+      name: this.state.name,
+      age: this.state.age,
+      height:this.state.height
+    };
```

Comment:
Because the only things in stays are these three key value pairs, you could shorten this like so, const smurfs = this.state